aptos-move/block-executor/src/proptest_types/bencher.rs

```diff
@@ -5,8 +5,8 @@
 use crate::{
     executor::BlockExecutor,
     proptest_types::types::{
-        EmptyDataView, ExpectedOutput, KeyType, Output, Task, Transaction, TransactionGen,
-        TransactionGenParams, ValueType,
+        EmptyDataView, EventType, ExpectedOutput, KeyType, Output, Task, Transaction,
+        TransactionGen, TransactionGenParams, ValueType,
```

Comment:
Why TransactionGen?

Reply:
I didn't add TransactionGen. Rust lint just moved it from the previous line.

Reply:
oh, sorry, lol